Formula/ghc.rb

```diff
@@ -40,34 +41,87 @@ class Ghc < Formula
     end
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/hjelmn/49f70df40da0399006031d854f73473e/raw/601bed7124f0fb9787d94806eab61645b85dab0a/ghc-8.10.3.patch"
```

Comment:
```suggestion
    url "https://raw.githubusercontent.com/Homebrew/formula-patches/1771693ca362a0f49281f92cc2042352834637f1/ghc/ghc-8.10.3.patch"
```
I think you should be able to use this URL now